exporter.go

```diff
@@ -11,18 +11,18 @@ import (
 	"strings"
 
 	"github.com/BurntSushi/toml"
+	"github.com/buildpack/lifecycle/img"
 	"github.com/google/go-containerregistry/pkg/v1"
 	"github.com/google/go-containerregistry/pkg/v1/mutate"
 	"github.com/pkg/errors"
-
-	"github.com/buildpack/lifecycle/img"
```

Comment:
Seems like a mistake?

Reply:
vim just merged all the imports together and sorted them. It seemed a good whitespace change so I left it in

Reply:
The import pattern we've used so far is:
```
<stdlib>

<external>

<internal>
```

Reply:
I would rather let go tooling do it's thing.

Reply:
I have disabled go tooling to edit the file and splt this back out